marklogic-data-hub/src/test/resources/temporal-test/step-definitions/custom/temporal-test/temporal-test.step.json

```diff
@@ -0,0 +1,9 @@
+{
+  "lang": "zxx",
+  "name": "temporal-test",
+  "type": "CUSTOM",
+  "options": {
+    "outputFormat": "json"
+  },
+  "modulePath": "/custom-modules/custom/temporal-step/main.sjs"
```

Comment:
This is not related, but pinging @fsnow and @ryanjdew and @bsrikan about this - I've wondered if step definitions should go away. They certainly seem like unnecessary work when making a custom step. The only thing of interest here is the modulePath, and if I'm making a custom step, it's easier to make one new file - the custom step with a modulePath in it - vs making two new files. 

Perhaps step definitions don't need to go away, but should just become optional. They are a useful way to refer to OOTB step modules... though arguably, just using the OOTB step modulePath would be just as easy. The indirection added by having a step definition name and type just doesn't seem to buy much. The odds of the module path changing seem as low as the odds of the step definition name/type changing, so tossing the OOTB module path into a step seems fine.